DataFormats/Scouting/src/classes_def.xml

```diff
@@ -26,7 +26,33 @@
   </class>
   <class name="ScoutingTrack" ClassVersion="2">
       <version ClassVersion="2" checksum="1342928770"/>
-</class>
+  </class>
+  <class name="Run3ScoutingCaloJet" ClassVersion="3">
```

Comment:
these are all new data formats, so they could start with `ClassVersion="2"` ?

Reply:
ok then let me change to 2.

I was checking, maybe it is outdated or I misunderstood https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning
"Version numbers 0, 1, and 2 have special meaning to ROOT and should not be used as version numbers. By convention, CMS will use 3 as the initial version number of a class which has never been stored before"